Hakuba/Source/Core/Hakuba.swift

```diff
@@ -351,9 +351,9 @@ extension Hakuba: UITableViewDelegate {
         return getCellmodel(at: indexPath)?.height ?? 0
     }
     
-    public func tableView(_ tableView: UITableView, estimatedHeightForRowAt indexPath: IndexPath) -> CGFloat {
-        return self.tableView(tableView, heightForRowAt: indexPath)
-    }
+//    public func tableView(_ tableView: UITableView, estimatedHeightForRowAt indexPath: IndexPath) -> CGFloat {
+//        return self.tableView(tableView, heightForRowAt: indexPath)
+//    }
```

Comment:
Please remove the commented out codes.
Could you describe the usage of UITableViewAutomaticDimension in README?